cmd/controller-manager/app/options/options.go

```diff
@@ -36,6 +36,7 @@ type ControllerManagerServer struct {
 }
 
 const defaultResyncInterval = 5 * time.Minute
+const defaultBrokerRelistInterval = 24 * time.Hour
```

Comment:
Default is 24hrs, but doc says 30 mins. Potentially surprising to people if they don't have a default set in the chart?